### PR TITLE
Add known bugs to play promise

### PR DESCRIPTION
**Pull request by beaufortfrancois**
R: @mounirlamouri
BUG: https://bugs.chromium.org/p/chromium/issues/detail?id=718647

### CODE REVIEW

File: src/content/en/updates/2016/03/play-returns-promise.md [hunk 1]
```diff
@@ -71,5 +71,15 @@ of this new functionality. View it in a browser such as Chrome 50 that supports
 interface. Be forewarned: the page will automatically play music when you visit it. (Unless, of
 course, it doesn’t!)
 
+## Relevant bugs {: #bugs }
```
Comment (mounirlamouri) [inline src/content/en/updates/2016/03/play-returns-promise.md]: I wouldn't call this a bug as it's actually part of the spec and there isn't much we can do. The bug will most likely be closed after this article is updated.

File: src/content/en/updates/2016/03/play-returns-promise.md [hunk 2]
```diff
@@ -71,5 +71,15 @@ of this new functionality. View it in a browser such as Chrome 50 that supports
...
+
+### `<source>` within `<video>` makes `play()` promise never rejects
+
+For `<video src="not-existing-video.mp4"\>`, the `play()` promise rejects as
+expected as the video doesn't exist. For `<video><source
+src="not-existing-video.mp4" type='video/mp4'></video>`, the `play()` promise
+never rejects. It only happens if there are no valid sources.
```
Comment (mounirlamouri) [inline src/content/en/updates/2016/03/play-returns-promise.md]: If we don't want to surface this as a bug, we might want to point to the spec?

File: src/content/en/updates/2017/06/play-request-was-interrupted.md [hunk 3]
```diff
@@ -2,7 +2,7 @@ project_path: /web/_project.yaml
 book_path: /web/updates/_book.yaml
 description: What is really happening with "DOMException: The play() request was interrupted"?
 
-{# wf_updated_on: 2017-06-16 #}
+{# wf_updated_on: 2018-01-24 #}
 {# wf_published_on: 2017-06-14 #}
```
Comment (mounirlamouri) [inline src/content/en/updates/2017/06/play-request-was-interrupted.md]: All comments above apply to this file.

**Comment by beaufortfrancois** [comment]
@mounirlamouri I've renamed it to "Danger zone"

**Comment by mounirlamouri** [comment]
Do you think it makes sense to leave the bugs as explicitly? I wonder if readers may still assume that it's a bug and will be fixed.

**Comment by beaufortfrancois** [comment]
I believe that is a way for us to see how many people will stumble upon this and might add their voice so that we can eventually "fix" the spec if needed.

**Comment by mounirlamouri** [comment]
I'm not sure we can just "fix" the spec but that's really your editorial call so LGTM :)

**Comment by beaufortfrancois** [comment]
@petele Can you merge and push live?

**Comment by petele** [comment]
will be pushed live tomorrow

**Pull request merged**